.travis.yml

```diff
@@ -57,7 +57,7 @@ cache:
         ccache: true
         pip: true
         directories:
-                - dpdk
+                #- dpdk
```

Comment:
I assume we'll turn this back on before merging if this fix works.

Reply:
+1